lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -0,0 +1,112 @@
+""" API v0 views. """
+import logging
+
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.lib.api import permissions
+from lms.djangoapps.certificates.api import get_certificate_for_user
```

Comment:
nitpick: alphabetize.   (not going to keep calling these out individually, but please fix other import organization issues in this PR)